packages/react-divider/Spec.md

```diff
@@ -0,0 +1,137 @@
+# Divider Spec
+
+## Background
+
+The divider control is to replace the Separator control currently in use. Its purpose is to visual separate content.
+Adding minor structural updates and new engineering practices and update the naming conventions to match naming conventions to OpenUI.
+
+## Prior Art
+
+[Github Epic Convergance Issue #16254](https://github.com/microsoft/fluentui/issues/16254)
+
+[Github Convergance Issue #15759](https://github.com/microsoft/fluentui/issues/15759)
+
+[Fluent UI Separator](https://developer.microsoft.com/en-us/fluentui#/controls/web/separator)
+
+[Open UI Component Matrix](https://open-ui.org/analysis/component-matrix)
+
+[Ant Design Divider](https://ant.design/components/divider/)
+
+[Fast Divider](https://explore.fast.design/components/fast-divider)
+
+[Semantic UI Divider](https://semantic-ui.com/elements/divider.html)
+
+## Comparison of v8 and v0
+
+- v8 - [Separator](https://developer.microsoft.com/en-us/fluentui#/controls/web/separator)
+- v0 - [Divider](https://fluentsite.z22.web.core.windows.net/0.51.4/components/divider/definition)
+
+## Sample Code
+
+Basic Examples:
+
+```html
+<Divider />
+<Divider vertical="true" />
+<Divider>This is a divider with content</Divider>
+```
+
+## Variants
+
+The divider will have two main variants, a horzontal and vertical render. The control is the same with the optional property `vertical="true"` defining the rendering type. As per current definition, content can only be added if the divider is horizontal (TBD).
+
+Standard default horizontal
+
+```html
+<Divider />
+```
+
+Vertical
+
+```html
+<Divider vertical="true" />
+```
+
+## API
+
+From [Divider.types.tsx](http://about:blank)
+
+### Slots
+
+- content (?)
+
+### Props
+
+```ts
+export type DividerProps = ComponentProps &
+  React.HTMLAttributes<HTMLElement> & {
+    /**
+     * A divider can justify the content. Center is default.
+     */
+    alignContent?: 'start' | 'end' | 'center';
+
+    children?: any;
+
+    /* A divider can have a overriding color */
+    color?: string;
```

Comment:
I think this should be a token since it only affects styling and nothing else.

Reply:
As per discussion this is will be left in until we have a token story